packages/web/src/components/track/desktop/ConnectedTrackTile.module.css

```diff
@@ -48,35 +48,8 @@
   opacity: 0.75;
 }
 
-.name,
-.name:hover,
-.name:active {
-  color: inherit;
-  text-decoration: inherit;
-}
-
-.userName {
-  display: inline-flex;
-  font-weight: var(--font-medium);
-  align-items: center;
-  /**
-   * Some webkit browsers (safari) have some trouble rendering this specific
```

Comment:
i imagine losing this ok, but we check safari just in case?

Reply:
just confirmed we chillin, good call